### ISSUE TITLE
Empty Pages: pgmpy.org/examples and pgmpy.org/tutorials

### ISSUE THREAD

**Issue by gweesip** [pgmpy/pgmpy#1707]
### Subject of the issue
Examples and tutorials are missing from the documentation, both pgmpy.org and local builds.

### Your environment
* pgmpy version: 0.1.23
* Python version: 3.8.10
* Operating System: Ubuntu 20.04.4 LTS
* Browsers: Firefox and Chrome

### Steps to reproduce
Go to website [examples page](https://pgmpy.org/examples.html) or [tutorials page](https://pgmpy.org/tutorial.html). See nothing :(

Same results with local build of documentation.

### Expected behaviour
Should see links to tutorials and examples in both navigation bar and main page. 

### Actual behaviour
![noexamples](https://github.com/pgmpy/pgmpy/assets/18489517/a5a75947-29a9-4e0c-8f0b-a4a554602a69)

**Comment by ankurankan** [comment]
@gweesip Thanks for reporting this. You should be able to see the links to the notebooks in the sidebar now. Adding links to the notebooks in the pages is turing out to be a bit tricker, and I haven't been able to figure out a way to do that yet.

**Comment by gweesip** [comment]
@ankurankan How did you fix the sidebars (looking to update on my local version as well)? Does it have anything to do with the sphinx output location? I noticed when following the documentation instructions from the readme, sphinx puts the output in ../../pgmpy_docs/html, not _build/html as specified in the readme.

AFAIK, the page has always been empty, but the sidebars going missing was a more recent issue. At least now there is way to access the notebooks on the site, so that's a win. Thanks!

**Comment by ankurankan** [comment]
@gweesip Yeah, a few things are hardcoded to my setup, that's why you are having issues with generating the docs locally. You would need the following this to make it work:
1. The `pgmpy_docs` dir should be at the same level as the main pgmpy dir.
2. You will need to symlink the notebooks in `pgmpy/examples` to `pgmpy/docs/examples` dir.
3. You will need to symlink the notebooks dir in pgmpy_notebook repository (`pgmpy_notebook/notebooks`) to `pgmpy/docs/detailed_notebooks` dir.

After this if you run `make html` from the pgmpy/docs dir, it should work.
